### DISCUSSION TITLE
nuxt 3 copy my public folder to .output

### DISCUSSION THREAD

**Discussion by cyperdark** [nuxt/nuxt#16263] [Questions]
Hello, i moving from nuxt 2 to nuxt 3.
And after running ``npm run build`` in nuxt 3, my ``/public`` folder copied to ``.output/public``, also if i add any files to this folder, i couldn't access them via url, is there a way to disable this behaviour?

**Comment by TheAlexLichter** [comment]
According to https://github.com/nuxt/framework/issues/9313 it is not possible to add files to the `public` folder after build. You can use Nuxt hooks to do that though ☺️

---

`public` in Nuxt3 is what `static` was in Nuxt2. You can rename the "public" folder to avoid that the files are being accessible.

---

Linking https://github.com/nuxt/nuxt/issues/19263 in favor.

**Reply by cyperdark** [reply]
yeah but it'll not update when i added a new file to this folder

**Reply by TheAlexLichter** [reply]
You cannot after build, see https://github.com/nuxt/framework/issues/9313.
